### ISSUE TITLE
new/delete mismatch

### ISSUE THREAD

**Issue by dimpase** [Singular/Singular#865]
https://github.com/Singular/Sources/blob/368dcf94cd3c437d819fd33a7de8c094b0cc24e1/libpolys/coeffs/shortfl.cc#L269

built with clang produces (different line numberings, as it's a bit older than this branch):
 
```
shortfl.cc:294:5: warning: 'delete' applied to a pointer that was allocated with 'new[]'; did you mean 'delete[]'? [-Wmismatched-new-delete]
    delete chbr;
    ^
          []
shortfl.cc:287:18: note: allocated with 'new[]' here
    char* chbr = new char[n+3];
                 ^
```

**Comment by fingolfin** [comment]
This was fixed in f2a4ddf13a67aded2d7eb6d181c2cfda9ceb300a, so this issue could be closed

**Comment by dimpase** [comment]
Thanks!
